adhocracy-plus/assets/scss/components/_a4-comments.scss

```diff
@@ -124,18 +124,34 @@
 
 .a4-comments__action-bar__btn {
     margin-left: 0.2 * $spacer !important;
+    text-decoration: underline;
```

Comment:
should we wrap the text (e.g. `report`) in a span and apply the underline there? Otherwise it looks a bit weird when applied to the whole anchor (i.e. looks like there is some padding missing between the icon and the text. Also not essential though if we want to get this merged